src/ImmersedBoundaries/active_cells_map.jl

```diff
@@ -103,9 +146,49 @@ end
 
 @inline add_3rd_index(t::Tuple, k) = (t[1], t[2], k) 
 
+active_cells_interior_map(ibg) = active_interior_indices(ibg; parameters = :xyz)
+
+# In case of a `DistributedGrid` we want to have different maps depending on the 
+# partitioning of the domain
+function active_cells_interior_map(ibg::ImmersedBoundaryGrid{<:Any, <:Any, <:Any, <:Any, <:DistributedGrid})
```

Comment:
I think this builds "`interior_active_cells`. So the name should be the same without moving the word `interior` around. Make it simple, always repeat the same phrase.